SalesforceFoundation/EDA
# Concealed learner visualizer prototype

# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes
* Edit any flexipage.
** Confirm that the course learner visualizer is not available.
* Run the config_dev flow.
** Confirm that the flow runs without errors.
** Confirm that the Contact object is not using a custom flexipage.
* Run the config_qa flow.
** Confirm that the flow runs without errors.
** Confirm that the Contact object is not using a custom flexipage.
* Run the learner_portal_setup flow.
** Confirm that the flow runs without errors.
